ci/socket-io-restart-url-auth.js

```diff
@@ -0,0 +1,40 @@
+const { Socket } = require("socket.io");
+
+let createServer = require("http").createServer;
+let server = createServer();
+const io = require("socket.io")(server);
+const port = 4206;
+const timeout = 200;
+
+function isTimestampValid(timestampStr) {
+  if (timestampStr === undefined) return false;
+  const timestamp = parseInt(timestampStr);
+  if (isNaN(timestamp)) return false;
+
+  const diff = Date.now() - timestamp;
+  return Math.abs(diff) <= 1000;
```

Comment:
Please add a comment on why this number is 1000 or store it in a `const` with a descriptive name.